xds/src/main/java/io/grpc/xds/LoadReportClient.java

```diff
@@ -392,93 +426,24 @@ public void run() {
           LoadReportingServiceGrpc.newStub(xdsChannel.getManagedChannel());
       lrsRequestWriterV3 = stubV3.withWaitForReady().streamLoadStats(lrsResponseReaderV3);
       logger.log(XdsLogLevel.DEBUG, "Sending initial LRS request");
-      sendLoadStatsRequest(new LoadStatsRequestData(node, null));
+      sendLoadStatsRequest(Collections.<ClusterStats>emptyList());
     }
 
     @Override
-    void sendLoadStatsRequest(LoadStatsRequestData request) {
-      LoadStatsRequest requestProto = request.toEnvoyProtoV3();
-      lrsRequestWriterV3.onNext(requestProto);
-      logger.log(XdsLogLevel.DEBUG, "Sent LoadStatsRequest\n{0}", requestProto);
+    void sendLoadStatsRequest(List<ClusterStats> clusterStatsList) {
+      LoadStatsRequest.Builder requestBuilder =
+          LoadStatsRequest.newBuilder().setNode(node.toEnvoyProtoNode());
+      for (ClusterStats stats : clusterStatsList) {
+        requestBuilder.addClusterStats(stats.toEnvoyProtoClusterStats());
+      }
+      LoadStatsRequest request = requestBuilder.build();
+      lrsRequestWriterV3.onNext(request);
+      logger.log(XdsLogLevel.DEBUG, "Sent LoadStatsRequest\n{0}", request);
     }
 
     @Override
     void sendError(Exception error) {
       lrsRequestWriterV3.onError(error);
     }
   }
-
-  private static final class LoadStatsRequestData {
```

Comment:
These two helper classes are redundant, they just repackage the fields in the proto messages/data structs. We can simply pass them as individual arguments to methods to be called.